### PR TITLE
Bump Bolt dependencies

### PR DESCRIPTION
**Pull request by lucywyman**
This bumps Bolt gem dependencies to their latest versions from the
Gemfile.lock.

### CODE REVIEW

**Comment by lucywyman** [comment]
M'k, I filed https://tickets.puppetlabs.com/browse/RE-13838 about the CI failures, but I don't think they started with this PR or should block it.

@puppetlabs/installer-and-management this updates:
- rubygem-bcrypt_pbkdf
- rubygem-multi_json

@puppetlabs/night-s-watch this updates:
- rubygem-concurrent-ruby
- rubygem-multi_json

Let me know if y'all have any concerns!

File: configs/components/rubygem-facter.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 component 'rubygem-facter' do |pkg, settings, platform|
-  pkg.version '4.0.47'
-  pkg.md5sum '30577ad8e59fb7d6c09dd53115b2a15d'
+  pkg.version '4.0.48'
```
Comment (gimmyxd) [inline configs/components/rubygem-facter.rb:2]: Facter 4.0.48 had a bug that was breaking some aggregate facts(FACT-2919).
It was fixed and released in 4.0.49

Comment (lucywyman) [reply configs/components/rubygem-facter.rb]: Sweet, updated.

**Comment by lucywyman** [comment]
Mmk, updated this to the latest versions:
@puppetlabs/installer-and-management this updates:
- rubygem-multi_json

@puppetlabs/night-s-watch this updates:
- rubygem-concurrent-ruby
 - rubygem-multi_json
 - rubygem-thor

Revisiting this - let me know if y'all have any concerns with updating any of the above gems or run into any trouble testing, otherwise :+1: and we can get these updated!

**Pull request closed without merging**